JASP-Engine/JASP/R/semsimple.R

```diff
@@ -207,6 +207,15 @@
                      USE.NAMES = FALSE)])
 }
 
+.decodeVarsInMessage <- function(encodedVars, message) {
```

Comment:
This is great for the problem at hand, but we would like to generalise this to other analyses don't we?

Reply:
Yeah, we could just do this in the setErrorMessage of the jaspObjects/results?
I'm pretty sure we won't find anything else in there that will accidentally transforms into a known columnname after decodeing the base64...